src/app/plan/components/Wizard/WizardComponent.tsx

```diff
@@ -241,6 +287,10 @@ const WizardComponent = (props: IOtherProps & FormikProps<IFormValues>) => {
       //update plan & start status polling on results page
       planUpdateRequest(props.values, false);
     }
+    if (prev.prevId === stepId.Hooks && curr.id === stepId.StorageClass) {
```

Comment:
Another use case for https://github.com/patternfly/patternfly-react/issues/4104, we wouldn't need this `isAddHooksOpen` state if we just had `currentStep` lifted out of `Wizard`.